chargeio/services/ServiceBase.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ChargeIO.Infrastructure;
+
+namespace ChargeIo.services
```

Comment:
This namespace is inconsistent with the rest of the library -- it should be "ChargeIO.Services".